core/include/webview/detail/backends/win32_edge.hh

```diff
@@ -412,9 +412,6 @@ public:
       on_window_created();
 
       m_dpi = get_window_dpi(m_window);
-      constexpr const int initial_width = 640;
-      constexpr const int initial_height = 480;
-      set_size(initial_width, initial_height, WEBVIEW_HINT_NONE);
```

Comment:
This is not enough to prevent the "jump" on windows. The initial size is still being set here: https://github.com/webview/webview/blob/e2bd2e55c5984e43ebb3cc92fe1ebcb727917cb2/core/include/webview/detail/backends/win32_edge.hh#L407-L408

Reply:
Mmmmm...
Looks like MS requires values for some of those: https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-createwindoww

Strange though, should be `0` width `0`height, ie. not there...

I am not in a Windows dev env ATM, perhaps you could fiddle around with the parameters and report back if there is a magic formula that prevents the glitch?

Currently `CreateWindowW` is called in the class constructor, so we can't pass user values for width and height. Moving it out of the class constructor is probably going to pull on a loose thread that we don't want to pull.

@SteffenL @phildremi Any insights?

Reply:
> Strange though, should be 0 width 0height, ie. not there...

It still renders the toolbar, so you briefly see a toolbar with no window "body"

>  perhaps you could fiddle around with the parameters and report back if there is a magic formula that prevents the glitch

The only reliable thing I've found is to set the initial position size in `CreateWindowW`. So we'd either need to defer that to `run` (which, as you noted, could potentially be a breaking change), or have a way to pass the position/size to the `webview` constructor.

Or, as a hacky approach, initially pass some very large negative values for x/y so the window renders offscreen until `set_size` is called.

Reply:
@rcorre I invited you to be a collaborator on my Webview fork. You should have full access to the `mac-default-size` branch (this PR).

Reply:
@rcorre I updated this PR by moving `ShowWindow`, `UpdateWindow` and `SetFocus` to the end of `set_size_impl` (with a backstop) in `win32_edge_engine`. CI tests are passing, but would you please confirm if this visually works and resolves the glitch?

Reply:
Thanks @citkane! Your latest patch works for me. I missed that webview was calling `ShowWindow` in the constructor. Moving that to `run` means I can resize/reposition the window without any jumps.